app/views/users/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="users-container">
+  <% if @users.any? %>
+    <div class="post-header">People you may know:</div>
+    <%= render 'users_list', users: @users %>
+  <% end %>
```

Comment:
This page should be displayed in the interface with a "request friendship" button, so we can invite other users in the social media application

Reply:
hi samgaco did you notice it was the first version of the friendship1 ?? the point is to set of the backend in friendship v1 and get it working not to implement notifications that will be done in friendship v2

Reply:
I suggest you take a look at what you are  supposed to be reviewing before you leave a comment next time

Reply:
hi, Samuel, I got your review on my pull request.
by any chance did you check what you were supposed to be reviewing?
cause if you did you probably would not leave the comment you left and would have noticed that milestone 5 has nothing to do with your "supposed " requirement. setting up the back-end and integration test is what is needed in this milestone while setting it up in the user interface is what milestone 6 is all about.
here's the link in case you get confused next time
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/